openpype/hosts/maya/startup/userSetup.py

```diff
@@ -29,18 +29,19 @@
 
 
 # Build a shelf.
-settings = get_project_settings(os.environ['AVALON_PROJECT'])
+project_name = get_current_project_name()
+settings = get_project_settings(project_name)
 shelf_preset = settings['maya'].get('project_shelf')
 
 if shelf_preset:
-    project = os.environ["AVALON_PROJECT"]
-
-    icon_path = os.path.join(os.environ['OPENPYPE_PROJECT_SCRIPTS'],
-                             project, "icons")
+    icon_path = os.path.join(
+        os.environ['OPENPYPE_PROJECT_SCRIPTS'],
+        project_name,
+        "icons")
     icon_path = os.path.abspath(icon_path)
 
     for i in shelf_preset['imports']:
-        import_string = "from {} import {}".format(project, i)
+        import_string = "from {} import {}".format(project_name, i)
```

Comment:
Import from project name, what? What's this demon logic here.

Reply:
I don't know, and I promissed myself that I won't care in this PR.